### ISSUE TITLE
sbt-scalafix still uses old semanticdb

### ISSUE THREAD

**Issue by avdv** [scalacenter/scalafix#1109]
Hi.

I noticed that sbt-scalafix 0.9.15-1 was released which, apparently, supports Scala 2.13.2 as it depends on scalameta 4.3.10.

Surprisingly, this was not the case, using the new plugin still reported `org.scalameta:semanticdb-scalac:4.3.8` for `scalafixSemanticdb.value`.

I see that the version is read from the `scalafix-interfaces.properties` file in BuildInfo, which still happens to point to version 0.9.15... :thought_balloon:

**Comment by mrdziuban** [comment]
I'm running into this as well. I can work around it by explicitly depending on a newer version, e.g.

```scala
addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "4.3.10" cross CrossVersion.full)
```

but it would be nice to have `addCompilerPlugin(scalafixSemanticdb)` work as expected

**Comment by avdv** [comment]
Yeah, you also can work around using
```
addCompilerPlugin(scalafixSemanticdb("4.3.10"))
```
I am just wondering whether this is working as intended?!

**Comment by bjaglin** [comment]
> I noticed that sbt-scalafix 0.9.15-1 was released which, apparently, supports Scala 2.13.2 as it depends on scalameta 4.3.10.

Actually https://github.com/scalacenter/sbt-scalafix/commit/7a30c70c28e7332648b13d11357a230b0ad73a8b is only used as a fallback. As sbt-scalafix currently depends on scalafix-interfaces 0.9.15, which still depends on scalameta 4.3.8, that version gets picked.

Once a new scalafix release and a corresponding sbt-scalafix release are cut, the work around above won't be needed any longer.

**Comment by bjaglin** [comment]
```
$ cs resolve --sbt-plugin ch.epfl.scala:sbt-scalafix:0.9.16
ch.epfl.scala:sbt-scalafix;sbtVersion=1.0;scalaVersion=2.12:0.9.16:default
ch.epfl.scala:scalafix-interfaces:0.9.16:default
com.geirsson:coursier-small_2.12:1.3.3:default
com.googlecode.javaewah:JavaEWAH:1.1.6:default
com.jcraft:jsch:0.1.55:default
com.jcraft:jzlib:1.1.1:default
io.get-coursier:interface:0.0.22:default
org.bouncycastle:bcpg-jdk15on:1.61:default
org.bouncycastle:bcpkix-jdk15on:1.61:default
org.bouncycastle:bcprov-jdk15on:1.61:default
org.eclipse.jgit:org.eclipse.jgit:5.5.1.201910021850-r:default
org.scala-lang:scala-library:2.12.6:default
org.slf4j:slf4j-api:1.7.2:default
```
